WooCommerce/Classes/ViewRelated/CardPresentPayments/CardReaderConnectionController.swift

```diff
@@ -33,6 +33,11 @@ final class CardReaderConnectionController {
         ///
         case connectToReader
 
+        /// A failure occurred while connecting. The search may continue or be canceled. At this time we
+        /// do not present the detailed error from the service.
+        ///
+        case connectingFailed
```

Comment:
I agree with presenting more actionable dialogs, but I think extending and testing with these SDK 2 messages should wait for a later PR (it will be hard to test that we got-it-right with SDK 1). How about I open a separate issue for that?

Reply:
On further discussion, I'll add an associated value we can leverage later.

Reply:
Added associated value in https://github.com/woocommerce/woocommerce-ios/pull/4957/commits/0db9ef0b4bc11efbf1857839a089867dc84f323f